src/Datadog.Trace.ClrProfiler.Managed/CallTarget/Handlers/IntegrationOptions.cs

```diff
@@ -17,10 +17,10 @@ internal static class IntegrationOptions<TIntegration, TTarget>
         internal static void DisableIntegration() => _disableIntegration = true;
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal static void LogException(Exception exception, [CallerLineNumber] int sourceLine = 0, [CallerFilePath] string sourceFile = "")
+        internal static void LogException(Exception exception)
```

Comment:
For my education, what was the result of aggressive inlining AND the caller line number/file?

Reply:
I removed the `CallerLineNumber` and `CallerFilePath` because this method is only called by the CallTargetInvoker, and knowing those two values is not relevant, also allows me remove both from the stack when calling this function.

For the `AggresiveInlining` this tell the jit to bypass the profitability analysis of the IL body and add it directly to a inlining candidate. Because the method is short and doesn't have any exception handler clause is indeed a good candidate for inlining.

According to AndyAyers: ```Aggressive inline candidates are assumed to be profitable; they bypass the normal profitability checks. So in particular we won't analyze their IL for patterns that might be optimizable.```

I think we should start using this (because we are enabling method inlining) on known hotpaths.

A single search into the runtime library can show you is used a lot 😂: https://github.com/dotnet/runtime/search?l=C%23&q=aggressiveinlining&type=code